app.py

```diff
@@ -58,30 +59,74 @@ def fn(self, arg):
 
 #This class ties all the docopt calling functions
 class ScreenOut (cmd.Cmd):
-    #The prompt that shows the use that he or she is running form the application in the cmd
+
+     #The below statements print a menu to act as a directive to the user
+    cprint("* * * * * * * * * * * * * * * * * * * * * * * * *",'cyan', attrs=['bold'])
```

Comment:
Love the UI you're going for. You might consider adding the stars in a neater way by doing cprint("*"*25) or the number of stars you want instead of typing them all out in your code.